src/js/features/repoLookup/repoLookup.comms.js

```diff
@@ -0,0 +1,27 @@
+import { githubApiAccessToken } from '../../../config/constants';
+
+const gitUrl = 'https://api.github.com/users';
+const accessToken = `?access_token=${githubApiAccessToken}`;
+
+export default class repoLookupComms {
+    static getUserRepos(username) {
+        return fetch(`${gitUrl}/${username}/repos${accessToken}`).
+            then(res => this.processJson(res)).
+            then(rawRepos => rawRepos.map(
+            repo => _.pick(repo, ['name', 'html_url'])
+        ));
+    }
+
+    static getUserData(username) {
+        return fetch(`${gitUrl}/${username}${accessToken}`).
```

Comment:
Tell the linting to shut up about having the dot at the end and not at the beginning :)